src/rpc/protocol.cpp

```diff
@@ -5,6 +5,7 @@
 
 #include "rpc/protocol.h"
 
+#include "main.h"
```

Comment:
The `include` added here can be removed.

Reply:
Agree, done!